src/resources/views/crud/columns/url.blade.php

```diff
@@ -0,0 +1,7 @@
+@php
```

Comment:
Excelleeent. The text gets clipped, but the URL does not, so the link still works. Great job on this one 👏👏👏